whitepaper/xmr-btc.tex

```diff
@@ -167,14 +167,14 @@ \subsubsection{\texttt{secp256k1}} as
 \section{Protocol}
 %In this chapter we describe the protocol for Alice and Bob to swap their funds atomically and without trusting each other.
 
-The overall protocol is as follows: Alice moves the monero into an address where each participant controls half of the private spend key. The Bitcoin scripting language is then used to reveal one of the halfs of the private spend key depending on which participant claims the bitcoin. Depending on who reveals the key share, the locked monero changes ownership. Bitcoin transactions are designed in such a way that if a participant follows the protocol he can't terminate with a loss.
+The overall protocol is as follows: Alice moves the monero into an address where each participant controls half of the private spend key (later referred to as key ``shares''). The Bitcoin scripting language is then used to reveal one of the shares depending on which participant claims the bitcoin. Depending on who reveals their half of the private spend key, the locked monero changes ownership. Bitcoin transactions are designed in such a way that if a participant follows the protocol they can't terminate with a loss.
```

Comment:
Note: Apparently this is the correct way to **``quote''** in Latex per https://tex.stackexchange.com/a/10672